apiv2/schema/schema.yaml

```diff
@@ -2322,11 +2261,14 @@ classes:
         range: string
         required: true
         description: Name of software used for reconstruction
-      is_canonical:
-        name: is_canonical
-        description: whether this tomogram is canonical for the run
-        from_schema: cdp-dataset-config
+      is_portal_standard:
+        description: whether this tomogram adheres to portal standards
         range: boolean
+        ifabsent: false
+      is_author_submitted:
+        description: whether this tomogram was submitted by the author of the dataset it belongs to.
+        range: boolean
+        ifabsent: false
```

Comment:
I believe this table also needs `is_visualization_default`, `publications` and `related_database_entries` fields.

Reply:
what does`is_visualization_default` mean for a tomogram? is that the default for an alignment or a run?